documentation/config_file_auth.md

```diff
@@ -23,7 +23,7 @@ However, Puppet Server now has its own `auth.conf` file that uses a new HOCON fo
 >
 > Puppet Server follows the following logic when determining whether to use the new or old authorization methods:
 >
-> * Requests to Puppet master service endpoints already manageable through the deprecated authorization methods and [Puppet `auth.conf`][] file --- such as `catalog`, `node`, and `report` --- use Puppet Server's new `auth.conf` rules **only** if the `use-legacy-auth-conf` setting in `puppet-server.conf` is set to `false`. If `use-legacy-auth-conf` is set to true (which is its default), Puppet Server warns you that the legacy authentication method is deprecated.
+> * Requests to Puppet master service endpoints already manageable through the deprecated authorization methods and [Puppet `auth.conf`][] file --- such as `catalog`, `node`, and `report` --- use Puppet Server's new `auth.conf` rules **only** if the `use-legacy-auth-conf` setting in `puppetserver.conf` is set to `false`. If `use-legacy-auth-conf` is set to true (which is its default), Puppet Server warns you that the legacy authentication method is deprecated.
```

Comment:
The name of this file in OSS - for some number of releases - been "puppetserver.conf".  See https://github.com/puppetlabs/puppetserver/blob/puppet-server-2.3.1/resources/ext/config/conf.d/puppetserver.conf.  So no change here I don't think.